client/login/magic-login/style.scss

```
@@ -868,3 +872,59 @@
 	margin: 0 auto;
 }
 /* stylelint-enable scales/font-weights */
+
+.card.magic-login__verify-code-form {
+	--magic-login-error-color: #cc1818;
+	padding: 0;
+	box-shadow: none;
+	border: none;
+}
+
+.magic-login__verify-code-field-container {
+	display: flex;
+	gap: 16px;
+	justify-content: space-between;
+	margin-top: 45px;
+	margin-bottom: 32px;
+
+	.form-text-input.magic-login__verify-code-character-field {
+		width: 61px;
+		height: 64px;
+		text-align: center;
+		padding: 0;
+		border: 1px solid var(--color-neutral-5, #dcdcdc);
+		border-radius: 4px;
+		@include heading-2x-large;
+
+		&:first-child {
+			margin-left: 0;
```

Let us use [RTL aware](https://hacks.mozilla.org/2015/10/building-rtl-aware-web-apps-websites-part-2/) alternative by making use of [Logical Properties](https://www.w3.org/TR/css-logical-1/).
```suggestion
			margin-inline-start: 0;
```

---

Fixed in ef18045